docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -230,7 +291,56 @@ can be imported by adding the following line to a PowerShell module:
     #Requires -Module Ansible.ModuleUtils.Legacy
 
 This will import the module_util at ``./lib/ansible/module_utils/powershell/Ansible.ModuleUtils.Legacy.psm1``
-and enable calling all of its functions.
+and enable calling all of its functions. As of Ansible 2.8, Windows module
+utils can also be written in C# and stored at ``lib/ansible/module_utils/csharp``.
```

Comment:
This bit should move up above lines 277-279 - otherwise we're telling people about referencing other C# utilities when they don't know C# utilities are possible . . .

Reply:
There's already a brief snippet around C# utils at https://github.com/ansible/ansible/pull/44705/files#diff-3caf171979d08135734172ec792f5820R176 which is above all this.